packages/mantine/src/components/prompt/Prompt.tsx

```diff
@@ -17,6 +17,7 @@ const useStyles = createStyles((theme) => ({
         borderBottom: `1px solid ${theme.colors.gray[3]}`,
         fontSize: theme.headings.sizes.h3.fontSize,
         lineHeight: theme.headings.sizes.h3.lineHeight,
+        wordWrap: 'break-word',
```

Comment:
According to MDN's [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), `word-wrap: break-word` is deprecated and can be replaced with `overflow-wrap: anywhere`.